pandas/sparse/array.py

```diff
@@ -59,7 +59,12 @@ def wrapper(self, other):
 
 
 def _sparse_array_op(left, right, op, name):
-    if np.isnan(left.fill_value):
+    if (np.isnan(left.fill_value) and np.isnan(right.fill_value) and
```

Comment:
so this begs the question of why we don't just always pass the fill values to the ufuncs (e.g. ``sparse_sub`` etc), which can then decide (based on ``np.isnan`` or (``isnull`` better)) on left and/or rhs whether to use it?